tests/integration/components/o-s-s/infinite-select-test.js

```diff
@@ -27,6 +27,27 @@ module('Integration | Component | o-s-s/infinite-select', function (hooks) {
   setupRenderingTest(hooks);
   setupIntl(hooks);
 
+  test('the index values from the @items array are available in the option named block', async function (assert) {
+    this.items = FAKE_DATA;
+    this.checkedIndex = 0;
+    this.onSelect = () => {};
+
+    await render(
+      hbs`
+         <OSS::InfiniteSelect
+              @items={{this.items}} @searchEnabled={{false}}
+              @onSelect={{this.onSelect}} >
+            <:option as |item index|>
+              {{item.name}}
+              <div class="index">{{index}}</div>
+            </:option>
+          </OSS::InfiniteSelect>
```

Comment:
Nit: indentation as usual :D

Reply:
```handlebars
<OSS::InfiniteSelect @items={{this.items}} @searchEnabled={{false}} @onSelect={{this.onSelect}}>
  <:option as |item index|>
    {{item.name}}
    <div class="index">{{index}}</div>
  </:option>
</OSS::InfiniteSelect>
```
or
```handlebars
<OSS::InfiniteSelect @items={{this.items}} @searchEnabled={{false}}
                     @onSelect={{this.onSelect}}>
  <:option as |item index|>
    {{item.name}}
    <div class="index">{{index}}</div>
  </:option>
</OSS::InfiniteSelect>
```
or
```handlebars
<OSS::InfiniteSelect
  @items={{this.items}}
  @searchEnabled={{false}}
  @onSelect={{this.onSelect}}
>
  <:option as |item index|>
    {{item.name}}
    <div class="index">{{index}}</div>
  </:option>
</OSS::InfiniteSelect>
```

Reply:
example of alignment with mutliple lines on the carousel [here](https://github.com/upfluence/oss-components/blob/master/tests/integration/components/o-s-s/carousel-test.ts#L20)